tools/codegen/dest/register_dispatch_key.py

```diff
@@ -489,21 +529,26 @@ def gen_class_ctor(self, k: SchemaKind, class_name: str, returns: int) -> str:
         elif k is SchemaKind.out:
             out_args = ', '.join(f"Tensor& out{i}" for i in range(returns))
             out_refs = ', '.join(f"std::ref(out{i})" for i in range(returns))
-            return f"{class_name}({out_args}) : outputs_{{ {out_refs} }} {{}}"
+            nullopts = ", ".join("c10::nullopt" for _ in range(returns))
+            return f"{class_name}({out_args}) : outputs_{{ {out_refs} }}, proxy_outputs_{{ {nullopts} }} {{}}"
```

Comment:
surprised this doesn't just default initialize appropriately

Reply:
Hmmm. You are right! It does initialize appropriately. Will fix it.